src/hist1d.jl

```diff
@@ -11,17 +11,13 @@ Base.lock(h::Hist1D) = lock(h.hlock)
 Base.unlock(h::Hist1D) = unlock(h.hlock)
 
 """
-    sample(h::Hist1D)
-    sample(h::Hist1D, n::Int)
+    sample(h::Hist1D, n::Int=1)
 
-Sample a histogram's with weights equal to bin count, one or `n` times.
-The returned sample value will be one of the bin's left edge.
+Sample a histogram's with weights equal to bin count, `n` times.
+The sampled values are the bin centers.
```

Comment:
this is not how ROOT does it, IIRC ROOT always give you bin's lower edge

Reply:
Ah ok let me modify it then